### PR TITLE
Lesson4

### PR DESCRIPTION
**Pull request by osipovichka**

### CODE REVIEW

File: src/main/java/com/example/blog/controllers/BlogController.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class BlogController {
+
+    @Autowired
+    private  PostRepository postRepository;
```
Comment (osipovik) [inline src/main/java/com/example/blog/controllers/BlogController.java]: remove extra space after modificator

File: src/main/java/com/example/blog/models/Post.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import javax.persistence.Id;
+
+@Entity
+public class Post {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    private String title, anons, full_text;
```
Comment (osipovik) [inline src/main/java/com/example/blog/models/Post.java]: Split declaration on separate lines

Comment (osipovik) [inline src/main/java/com/example/blog/models/Post.java]: rename `full_text` using camel case

Comment (osipovik) [reply src/main/java/com/example/blog/models/Post.java]: Add `lombok` to project

**Review by osipovik — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/blog/controllers/BlogController.java (hunk 1 above)
Comment (osipovichka) [reply src/main/java/com/example/blog/controllers/BlogController.java]: done sir

File: src/main/java/com/example/blog/models/Post.java (hunk 2 above)
Comment (osipovichka) [reply src/main/java/com/example/blog/models/Post.java]: done sir

**Pull request merged**